code/widgets/org.eclipse.scout.jswidgets.ui.html/src/main/js/jswidgets/table/TableForm.json

```diff
@@ -64,6 +64,14 @@
                   "text": "Html Column",
                   "htmlEnabled": true,
                   "width": 110
+                },
+                {
+                  "id": "HtmlTooltipColumn",
+                  "objectType": "BooleanColumn",
+                  "text": "Html Tooltip Column",
```

Comment:
Please don't add this column. Currently the columns show the different column types, the new column is a boolean column which is already there. If you want to test it we can add a setter on Column.js to dynamically change the tooltip text and header enabled flag.

Reply:
I just added the setters setHeaderTooltipText and setHeaderTooltipHtmlEnabled to Column.js.

Reply:
okay, removed :)